sensor.go

```diff
@@ -44,33 +44,36 @@ func (r *sensorS) setOptions(options *Options) {
 	}
 }
 
-func (r *sensorS) getOptions() *Options {
-	return r.options
-}
-
 func (r *sensorS) configureServiceName() {
 	if r.options != nil {
 		r.serviceName = r.options.Service
 	}
 
+	if r.serviceName == "" {
+		r.serviceName = os.Getenv("INSTANA_SERVICE_NAME")
+	}
+
```

Comment:
One change: The env var should override all.  then `options.Service` and the fallback.

Reply:
Currently it already overrides the fallback and my intention was that if something is configured explicitly in the code, it should take the precedence. However if we consider a use case when we need to rename a service without having to change the code, then an env var value prioritized over the option one makes total sense to me. Will adjust 👍